### DISCUSSION TITLE
xxx file not found in cpp project

### DISCUSSION THREAD

**Discussion by ViNull008** [ayamir/nvimdots#1264] [Q&A] [answered]
Open a cpp project folder, choose a file a.cpp, it wil show lots error such as 
![image](https://github.com/ayamir/nvimdots/assets/11844999/c157a038-61d0-4010-862a-5e0c364370bd)
some head files are even in same poroject ,it still show  "not found". 
std is cannot be recognized also
![image](https://github.com/ayamir/nvimdots/assets/11844999/c59c1a53-56e9-47a8-97f5-92e0f005329b)

Not sure it's can not be supported or my configure is error.
Some guy could tell me about it?
Thank you!

**Comment by Cyberczy** [comment, marked as answer]
https://github.com/ayamir/nvimdots/issues/370
I hope this is helpful for you.

**Reply by ViNull008** [reply]
Thank you !!
